Tests/integration_tests/e2e_tests_utils.py

```diff
@@ -0,0 +1,43 @@
+
+import subprocess
+import sys
+
+import demisto_client
+from demisto_sdk.commands.common.tools import (get_demisto_version,
+                                               )
+
+
+class e2e_tests_utils:
```

Comment:
no need for a class

Reply:
[Done](https://github.com/demisto/content/pull/20373/commits/5aa2261ff79dea6b427eb4870b9e9bd6db87cc96).